content/webapp/services/prismic/transformers/json-ld.ts

```diff
@@ -15,17 +15,28 @@ import { Exhibition } from '../../../types/exhibitions';
 import { ExhibitionGuide } from '../../../types/exhibition-guides';
 import { linkResolver } from '@weco/common/services/prismic/link-resolver';
 
+// Guide from schema.org
```

Comment:
Not sure Guide is the correct type either (sorry I probably wasn't clear in my previous comment). Or do you think it is?

Reply:
I read your previous comment too quickly, I thought you were suggesting 'Guide'. I'll have another dig through 'schema.org' and see if something fits better.